internal/services/engines/nodejs/rules_test.go

```diff
@@ -324,11 +324,6 @@ func TestRulesVulnerableCode(t *testing.T) {
 
 func TestRulesSafeCode(t *testing.T) {
 	testcases := []*testutil.RuleTestCase{
-		{
-			Name: "HS-JAVASCRIPT-2",
-			Rule: NewNoUseEval(),
-			Src:  SampleSafeHSJAVASCRIPT2,
-		},
```

Comment:
Shouldn't we test this scenario?

Reply:
Indeed, we now have safe cases. Adjusted!